docs/src/content/tutorial/boilerplate-project.md

```diff
@@ -61,9 +61,9 @@ npm run start
 
 Then open [http://127.0.0.1:3000/](http://127.0.0.1:3000/) in your browser and you should see this: ![](/front-5.png)
 
-Set your network in MetaMask to `127.0.0.1:8545`.
+If you are using MetaMask, set your network to `127.0.0.1:8545` and click the button in the web app. If you are using Coinbase Wallet, first click the button, then select the "Hardhat (localhost)" network in the wallet, and then click the button again.
```

Comment:
If the following updates to the boilerplate are good, it should auto-switch the user to `31337`:

https://github.com/NomicFoundation/hardhat-boilerplate/pull/90